app/data_migrations/populate_document_type.py

```diff
@@ -2,7 +2,7 @@
 
 from sqlalchemy.orm import Session
 
-from app.db.models.deprecated import DocumentType
+from app.db.models.lawpolicy import DocumentType
```

Comment:
nit: should this me `law_policy`?
```suggestion
from app.db.models.law_policy import DocumentType
```

Reply:
Happy to change if you want this

Reply:
if it's just a quick refactor it would be nice

Reply:
done